common/src/main/kotlin/action/Action.kt

```diff
@@ -104,4 +110,8 @@ open class Action(
         properties.add(property)
         return this
     }
+
+    open fun requiresEvent(): Boolean {
+        return false
+    }
```

Comment:
Would be good to have a blacklist/whitelist events similar to how I do it on playground-bukkit, not 100% necessary, but useful :)